storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2186,6 +2298,15 @@ dberr_t dict_load_tables_from_space_id(space_id_t space_id, THD *thd,
     uint32_t compressed_sdi_len = compressed_buf_len;
     uint32_t uncompressed_sdi_len = uncompressed_buf_len;
 
+    if (ib_key.sdi_key->type != 1 /* dd::Sdi_type::TABLE */) {
+      continue;
+    }
+
+    /* in case of duplicate sdi, sdi_id is update to correct sdi */
```

Comment:
```suggestion
    /* In case of duplicate SDIs, sdi_id is the latest id according
    to DD, so we skip other dd::Table SDIs in the IBD file */
```

Reply:
fixed